src/gui/plugins/plotting/Plotting.hh

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef IGNITION_GUI_PLUGINS_PLOTTING_HH_
+#define IGNITION_GUI_PLUGINS_PLOTTING_HH_
+
+#include <ignition/gui/qt.h>
+#include <ignition/gui/Application.hh>
+#include <ignition/gui/PlottingInterface.hh>
+#include <ignition/gazebo/gui/GuiSystem.hh>
+#include <ignition/math/Vector3.hh>
+#include <ignition/math/Pose3.hh>
+
+#include <map>
+#include <string>
+#include <memory>
+
+#include "ignition/gazebo/components/Actor.hh"
```

Comment:
These includes can all be in the `.cc` file. It's preferable to include on the cc file whenever possible, because they won't get added to everyone who includes this header.